recipe/migrations/pypy38.yaml

```diff
@@ -36,6 +36,8 @@ python:
   - 3.9.* *_73_pypy    # [not (osx and arm64)]
 numpy:
   # part of a zip_keys: python, python_impl, numpy
-  - 1.22               # [not (osx and arm64)]
+  - 2.0                # [not (osx and arm64)]
```

Comment:
Also, I think we may want to keep PyPy building against 1.x. In particular, it [looks like](https://github.com/conda-forge/scipy-feedstock/pull/274) there will be no SciPy builds for PyPy against NumPy 2.0 in the near future, so that means all dependents of SciPy would fail to migrate unless they too drop PyPy, which would be unnecessarily disruptive IMO.

Reply:
Since our main PyPy support explicitly [asked](https://github.com/conda-forge/scipy-feedstock/pull/274#issuecomment-2092041382) for this as well, I'm excluding PyPy from the NumPy 2 effort here.